src/places.js

```diff
@@ -12,13 +12,46 @@ import './places.scss';
 import clearIcon from './icons/clear.svg';
 import pinIcon from './icons/address.svg';
 
+const errors = {
+  multiContainers:
+`Algolia Places: 'container' must point to a single <input> element.
+Example: instantiate the library twice if you want to bind two <inputs>.
+
+See https://community.algolia.com/places/documentation.html#api-options-container`,
+  badContainer:
+`Algolia Places: 'container' must point to an <input> element.
+
+See https://community.algolia.com/places/documentation.html#api-options-container`
+};
+
 export default function places(options) {
   const {
     container,
     style,
     autocompleteOptions: userAutocompleteOptions = {}
   } = options;
 
+  // multiple DOM elements targeted
+  if (container instanceof NodeList && container.length > 1) {
+    throw new Error(errors.multiContainers);
+  }
+
+  // container sent as a string, resolve it for multiple DOM elements issue
+  if (typeof container === 'string') {
+    const resolvedContainer = document.querySelectorAll(container);
+    return places({container: resolvedContainer, ...options});
+  }
+
+  // if NodeList received, resolve to the first one
+  if (container instanceof NodeList) {
+    return places({container: container[0], ...options});
+  }
+
+  // if not an <input>, error
+  if (!(container instanceof HTMLInputElement)) {
```

Comment:
I think this is handled by the previous line where if I have a NodeList, I will call places again with the first element of the NodeList.